.github/workflows/audit.yml

```diff
@@ -0,0 +1,12 @@
+name: Audit
+
+on:
+  push:
+  schedule:
+    - cron: "40 10 * * *"
+
+jobs:
+    audit:
+      uses: inrupt/javascript-style-configs/.github/workflows/reusable-audit.yml@create-reusable-workflows
```

Comment:
Oh, also, these will break once that PR is merged, side-effect of using something in testing. Let's make sure these work then merge that branch